### PR TITLE
Enable scaling nodes through configuration

### PR DESCRIPTION
**Pull request by dfinteligenz**

### CODE REVIEW

File: engine/admin-api/domain/usecase/krt/parser/parser.go [hunk 1]
```diff
@@ -88,6 +88,14 @@ func (p *Parser) ParseKrtYaml(krtFilePath string) (*krt.Krt, error) {
 		return nil, fmt.Errorf("error on KRT Yaml parsing: %w", err)
 	}
 
+	p.logger.Infof("++++++++++++++++++++++++++++")
```
Comment (alexrodfe) [inline engine/admin-api/domain/usecase/krt/parser/parser.go:91]: we forgot about this logger

File: engine/admin-api/domain/usecase/version_interactor.go [hunk 2]
```diff
@@ -261,6 +261,8 @@ func (i *VersionInteractor) completeVersionCreation(
 	docFolder := path.Join(tmpDir, "docs")
 	contentErrors = i.saveKRTDoc(runtime.ID, docFolder, versionCreated, contentErrors, ctx)
 
+	i.logger.Infof("%v+", krtYml)
```
Comment (alexrodfe) [inline engine/admin-api/domain/usecase/version_interactor.go:264]: another logger

**Review by alexrodfe — CHANGES_REQUESTED** [review]
Great!
Just a quick cleanup and we should be done

File: engine/admin-api/domain/usecase/version_interactor.go (hunk 2 above)
Comment (dfinteligenz) [reply engine/admin-api/domain/usecase/version_interactor.go]: Fixed!

**Review by alexrodfe — APPROVED** [review]

**Pull request merged**